sdcm/gemini_thread.py

```diff
@@ -72,7 +72,7 @@ def gemini_result_file(self):
         return self._gemini_result_file
 
     def _generate_gemini_command(self):
-        seed = self.params.get('gemini_seed', None)
+        seed = self.params.get('gemini_seed')
```

Comment:
add to `test_default.yaml`

Reply:
a you sure we need it there? By test code seems we don't need the default value for this parameter